src/jarabe/desktop/schoolserver.py

```diff
@@ -56,7 +61,25 @@ def _generate_serial_number():
     return serial
 
 
-def _store_identifiers(serial_number, uuid_, backup_url):
+def _get_history_for_serial(serial_number):
+    identifier_path = os.path.join(env.get_profile_path(), 'identifiers')
+    if not os.path.exists(identifier_path):
+        os.mkdir(identifier_path)
+
+    file_path = os.path.join(identifier_path, 'server_history')
+    if os.path.exists(file_path):
+        json_file = open(file_path, 'r')
+        data = json.loads(json_file.read())
+        json_file.close()
+        if serial_number in data:
```

Comment:
These lines (join a path, check exists, open, load, close) are also closely related to the other lines you add below, please refactor into a read from a file in identifiers path, returning data.